atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/service/TransactionServices.java

```diff
@@ -15,15 +15,48 @@
  */
 package com.palantir.atlasdb.transaction.service;
 
+import com.google.common.collect.ImmutableMap;
+import com.palantir.atlasdb.coordination.CoordinationService;
+import com.palantir.atlasdb.internalschema.InternalSchemaMetadata;
+import com.palantir.atlasdb.internalschema.TransactionSchemaManager;
+import com.palantir.atlasdb.internalschema.persistence.CoordinationServices;
 import com.palantir.atlasdb.keyvalue.api.KeyValueService;
+import com.palantir.timestamp.TimestampService;
 
 public final class TransactionServices {
     private TransactionServices() {
-        // Uility class
+        // Utility class
     }
 
     public static TransactionService createTransactionService(
-            KeyValueService keyValueService) {
+            KeyValueService keyValueService, CoordinationService<InternalSchemaMetadata> coordinationService) {
```

Comment:
Will need to coordinate releases with other internal services/libraries that need a `TransactionService`

Reply:
Good point. I will mark this as a dev break (which it is)

Reply:
Also, if people are doing this then it probably makes sense for the `TransactionManager` to expose the `TransactionService` it uses